#Servo State

#Constants:
uint8 NOT_ACTIVE=0
uint8 ACTIVE=1

uint8 ActiveMode
uint8 Angle
uint8 Speed
